### PR TITLE
Applications: nrf5340_audio: Experimental addition of SDC in buildprog

### PR DESCRIPTION
**Pull request by koffes**
To be squashed

### CODE REVIEW

**Review by rick1082 — APPROVED** [review]
Confirmed this works, I'm using following command for testing:
$ python buildprog.py -c both -b debug -d both -p --pristine --ctlr="SDC" -f -s

But probably we could add a comment in "-h" for mentioning this feature?

File: applications/nrf5340_audio/tools/buildprog/buildprog.py [hunk 1]
```diff
@@ -299,22 +311,32 @@ def __main():
...
+        Controller.acs_nrf53,
     )
     parser.add_argument(
         "-m",
         "--mcuboot",
         required=("-M" in sys.argv or "--min_b0n" in sys.argv),
         choices=["external", "internal"],
         default='',
-        help="MCUBOOT with external, internal flash",
+        help="MCUBOOT with external, internal flash. Only for controller: "+Controller.acs_nrf53,
```
Comment (alexsven) [inline applications/nrf5340_audio/tools/buildprog/buildprog.py:339]: Nitpick: Could need some spaces around `+`

File: applications/nrf5340_audio/tools/buildprog/buildprog.py [hunk 2]
```diff
@@ -339,8 +361,16 @@ def __main():
         default=False,
         help="Set to generate a user specific Bluetooth device name. Note that this will put the computer user name on air in clear text",
     )
+
     options = parser.parse_args(args=sys.argv[1:])
 
+    if options.controller != Controller.acs_nrf53:
+        print(Fore.YELLOW +
+              "Experimental app - controller combination"+Style.RESET_ALL)
```
Comment (alexsven) [inline applications/nrf5340_audio/tools/buildprog/buildprog.py:369]:
```suggestion
              "Experimental app - controller combination" + Style.RESET_ALL)
```

**Review by alexsven — APPROVED** [review]

**Review by erikrobstad — CHANGES_REQUESTED** [review]
If there are no build folders from "previous SDC builds" present, this command does not work:
python buildprog.py -c net -b debug -d both -p --ctlr SDC --pristine
In other words, it is not possible to do '-c net'.
This should be improved, or possibly just e.g. throw a warning.
image

**Review by erikrobstad — APPROVED** [review]

**Pull request merged**